index.js

```diff
@@ -169,10 +170,10 @@ function cli(process_args) {
       dockerEvents: true,
       statsinterval: 30,
       add: [ 'host=' + os.hostname() ],
-      token: process.env.LOGENTRIES_TOKEN,
-      logstoken: process.env.LOGENTRIES_LOGSTOKEN || process.env.LOGENTRIES_TOKEN,
-      statstoken: process.env.LOGENTRIES_STATSTOKEN || process.env.LOGENTRIES_TOKEN,
-      eventstoken: process.env.LOGENTRIES_EVENTSTOKEN || process.env.LOGENTRIES_TOKEN,
+      token: process.env.INSIGHT_TOKEN,
+      logstoken: process.env.INSIGHT_LOGSTOKEN || process.env.INSIGHT_TOKEN,
```

Comment:
logstoken? should this be `INSIGHT_LOGTOKEN`?

Reply:
not fully sure what you mean, you mean the `logstoken:`? no, it's just the variable name that's used within the library

Reply:
just that "logs token" sounds wrong :p but it's consistent i guess